content/docs/for-app-developers/how-to/build-outputs/understand-failures.md

```diff
@@ -4,6 +4,6 @@ weight=99
 summary="How to troubleshoot when things go wrong."
 +++
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+While `Buildpacks` help developers transform applications' source code into container images that can run on any cloud, creating an error-free experience remains far from achieved.
 
-If you are familiar with this content and would like to make a contribution, please feel free to open a PR :)
+This guide explores some of the most common issues that may prevent image build completion and provides troubleshooting tips to help end-users navigate these issues.
```

Comment:
```suggestion
This guide catalogs some commonly reported issues that may prevent image build completion and provides troubleshooting tips to help end-users navigate these issues.

If you would like to report an issue, please open a PR against this page using the included template.
```

Reply:
I'll add a suggestion for the template :)

Reply:
@AidanDelaney WDYT of the above as a place to get started?

Reply:
Perfect! Better than what I was suggesting